src/OrchardCore/OrchardCore.Abstractions/Extensions/DocumentJsonSerializerOptionsConfiguration.cs

```diff
@@ -17,7 +16,19 @@ public DocumentJsonSerializerOptionsConfiguration(IOptions<JsonDerivedTypesOptio
 
     public void Configure(DocumentJsonSerializerOptions options)
     {
-        options.SerializerOptions.Merge(JOptions.Default);
```

Comment:
What's the wrong with `Merge`? if we really need this we could introduce an extension method if this could be used somewhere else

Reply:
Merge in this case could merge unwanted properties and we should only merge some of the properties on `JOptions.Base` not `JOptions.Default`. Currently the preview is un-usable and this fix is needed to make it reusable again.

Reply:
Which properties cause instability when overwritten? In my opinion using Merge and then explicitly restoring values that must be different from JOptions.Default would be more correct and more maintainable.

Reply:
@sarahelsaig there is a strange behavior with using `Merge`. If you try main now, you'll notice that all of the polymorphic json type are failing to convert. So Rule, Conditions, AdminNode, etc are are failing. So now the AdminMenu does not work, layers do not render correctly, etc.

Reply:
Yes this would also work. I actually tried that before. However, not all the properties we want to use in this case. `Encoder`, `NumberHandling`, or `AllowTrailingCommas`.

Reply:
I see. I still suggest using Merge and resetting those values explicitly.

But if you would rather stick with your current code, then please mention Encoder, NumberHandling, and AllowTrailingCommas in [the comment](https://github.com/OrchardCMS/OrchardCore/pull/16634/files#diff-6312ecbe30331924dc9a02f5024316f7fa1d4f18b58b9b46b5db3217a198b3efR19) by name for future reference.

Reply:
Its not just these 3 properties which is why I did not add them in the comment. I just stated `unwanted properties`. So in this case we explicitly set the properties that we want to include in the document serialize regardless to the logic in the merge. I do not want to revise that logic because the logic in merge could change overtime and I don't want that to impact this sensitive document serializer options.

Reply:
Ok, but if we are certain we don't want these 3 then you can mention them as examples: `(e.g. Encoder, NumberHandling, AllowTrailingCommas)`. It's better to have this information documented in code than to leave it up for the imagination when someone in the future reads that comment.

Reply:
added.